main/autogen/autogen.h

```diff
@@ -83,13 +83,18 @@ struct ParsedFile {
     std::string toString(core::Context ctx);
     std::string toMsgpack(core::Context ctx, int version);
     void classlist(core::Context ctx, std::vector<std::string> &out);
-    void subclasses(core::Context ctx, std::map<std::string, std::set<std::string>> &out);
```

Comment:
we try to call methods as actions.`listSubclasses`?
Also, from the current signature it's unclear for me `subclasses of what`, can we make it explicit in arguments?

Reply:
[fixed](https://github.com/sorbet/sorbet/pull/1040/files#diff-670beace6f2de3dc49c6286fb9310f27R89)